### PR TITLE
Add endpoint to fetch votes sent - Closes #1104

### LINKED ISSUE #1104: Add endpoint to fetch votes sent
### Description

With the DPoS module enabled, users should be able to see their outgoing votes.

### Acceptance Criteria

- The following endpoints are available
- Users are able to filter the tokens based on the query params listed below
- Integration tests are implemented

#### Endpoints
**HTTP**:  `GET /dpos/votes/sent`
**RPC**:    `get.dpos.votes.sent`

#### Request query params
Parameter | Type | Validation | Default | Comment
-- | -- | -- | -- | --
address | String | /^lsk([a-hjkm-z]\|[2-9]){38}$/ | (empty) | Required
name | String | /^[a-z0-9!@$&_.]{1,20}$/ | (empty) | 
limit | Number | <1;100> | 10 | 
offset | Number | <0;+Inf> | 0 |

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #1104 

### How was it solved?

- [x] The following endpoints are available:
  - [x] HTTP: GET /dpos/votes/sent
  - [x] RPC: get.dpos.votes.sent
- [x] Users are able to filter the tokens based on the specified query params
- [x] Update controller for votes sent
- [x] Refactor votes sent implementation
- [x] Add Integration tests

### How was it tested?
Local

### CODE REVIEW

File: services/blockchain-indexer/methods/dataService/controllers/dposVotesSent.js [hunk 1]
```diff
@@ -26,26 +25,15 @@ const {
 
 const getVotes = async params => {
 	const isFound = await confirmAnyId(params);
-	if (typeof params.anyId === 'string' && !params.address) return { status: NOT_FOUND, data: { error: `Account ${params.anyId} not found.` } };
 	if (!isFound && params.address) return { status: NOT_FOUND, data: { error: `Account ${params.address} not found.` } };
-	if (!isFound && params.username) return { status: NOT_FOUND, data: { error: `Account ${params.username} not found.` } };
-	if (!isFound && params.publicKey) return { status: NOT_FOUND, data: { error: `Account with a public key ${params.publicKey} not found.` } };
-	if (!isFound && params.secondPublicKey) return { status: NOT_FOUND, data: { error: `Account with a second public key ${params.secondPublicKey} not found.` } };
+	if (!isFound && params.name) return { status: NOT_FOUND, data: { error: `Account ${params.name} not found.` } };
```
Comment (sameersubudhi) [inline services/blockchain-indexer/methods/dataService/controllers/dposVotesSent.js:29]:
```suggestion
	if (!isFound && params.name) return { status: NOT_FOUND, data: { error: `Account with name ${params.name} not found.` } };
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

File: services/blockchain-indexer/shared/dataService/business/dpos/dposVotesSent.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	getHexAddressFromBase32,
+	getIndexedAccountInfo,
+} = require('../../../utils/accountUtils');
+const { getAddressByName } = require('../../../utils/delegateUtils');
+const { parseToJSONCompatObj } = require('../../../utils/parser');
+const { requestConnector } = require('../../../utils/request');
+
+const normalizeVote = vote => {
+	const normalizedVote = parseToJSONCompatObj(vote);
+	normalizedVote.delegateAddress = getBase32AddressFromHex(vote.delegateAddress);
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/dataService/business/dpos/dposVotesSent.js:29]: Don't we receive the address in the base32 format?

Comment (nagdahimanshu) [reply services/blockchain-indexer/shared/dataService/business/dpos/dposVotesSent.js:29]: No, we receive binary address from the node.

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Pull request merged**